tanner/emulators/php_code_injection.py

```diff
@@ -0,0 +1,36 @@
+import aiohttp
+import asyncio
+import logging
+
+from tanner.utils import patterns
+
+class PHPCodeInjection:
+    def __init__(self, loop=None):
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+        self.logger = logging.getLogger('tanner.php_code_injecton')
+
+    async def get_injection_result(self, code): 
+        code_injection_result = None
+        code = '<?php eval(\'$a = {code}\'); ?>'.format(code=code)
+        try:
+            async with aiohttp.ClientSession(loop=self._loop) as session:
+                async with session.post('http://127.0.0.1:8088/', data=code) as resp:
```

Comment:
Yeah, we can make the same change in `rfi` also where phpox address is written directly.